scripts/zones/Vunkerl_Inlet_[S]/npcs/qm4.lua

```diff
@@ -0,0 +1,32 @@
+-----------------------------------
+-- Area: Vunkerl_Inlet_[S]
+--  NPC: qm4 (???)
+-- Involved In Quest: REDEEMING_ROCKS
+-- !pos -412 -29 -45 83
+-----------------------------------
+local ID = require("scripts/zones/Vunkerl_Inlet_[S]/IDs")
+require("scripts/globals/keyitems")
+require("scripts/globals/quests")
+-----------------------------------
+
+function onTrade(player, npc, trade)
+end
+
+function onTrigger(player, npc)
+    local RedeemingRocks = player:getQuestStatus(CRYSTAL_WAR, tpz.quest.id.crystalWar.REDEEMING_ROCKS)
+    local RocksProg = player:getCharVar("RedeemingRocksProg")
```

Comment:
You can delete these two variables as they are only called once each in the function.

`if player:getQuestStatus(CRYSTAL_WAR, tpz.quest.id.crystalWar.REDEEMING_ROCKS) == QUEST_ACCEPTED and player:getCharVar("RedeemingRocksProg") == 3 then`

Also as per the style guide you want to eliminate parenthesis from your if statements unless they are used for order of operations.